src/views/collection-view/helpers/generate-collection-sidebar-nav-data.tsx

```diff
@@ -0,0 +1,32 @@
+import { ProductData } from 'types/products'
+import { CollectionViewSidebarContent } from 'components/tutorials-sidebar'
+import { CollectionCategorySidebarSection } from './format-sidebar-sections'
+
+/**
+ * Generate the Tutorials Collection sidebar nav data level.
+ * This is used both on the Collection view,
+ * and in the mobile menu on the individual Tutorials view
+ */
+export function generateCollectionSidebarNavData(
+	product: ProductData,
+	sidebarSections: CollectionCategorySidebarSection[]
+) {
+	return {
+		levelButtonProps: {
+			levelUpButtonText: `${product.name} Home`,
+			levelDownButtonText: 'Previous',
+		},
+		backToLinkProps: {
+			text: `${product.name} Home`,
+			href: `/${product.slug}`,
+		},
+		title: 'Tutorials',
+		visuallyHideTitle: true,
```

Comment:
💭 It might be a bit confusing tracking down how the themed title is generated. What have been your thoughts on the data driving the themed title item versus automating it at the component level? IOW if we defined the themed title here instead of lower down.

Reply:
I dig this! I'll give it a shot 👍

Reply:
Ah wait so I misinterpreted this - I'm actually not sure I follow. Currently, for collection sidebar levels (as used on product-tutorials views, collection views, and tutorial views for the mobile sidebar), we render the themed title item through the `CollectionViewSidebarContent` component.

Is there somewhere else we want to move that rendering? I guess we could pass it here as a first item in `children`, rather than within `<CollectionViewSidebarContent />`?

(I hadn't given it too much thought, `CollectionViewSidebarContent` just seemed like the most intuitive place at first!)

Reply:
Let's roll with what you have!